spec/lib/danger/commands/init_helpers/interviewer_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rspec'
+require 'danger/commands/init_helpers/interviewer'
+
+RSpec.describe Danger::Interviewer do
+  let(:cork) { double('cork') }
+  let(:interviewer) { Danger::Interviewer.new(cork) }
+
+  describe '#link' do
+    before do
+      allow(interviewer).to receive(:say)
+    end
+
+    it 'link URL is decorated' do
+      interviewer.link('http://danger.systems/')
```

Comment:
I personally prefer to declare this to `let` scope.
Bacause, when specs are added, we can reuse it.